web/modules/custom/joinup_migrate/joinup_migrate.post_update.php

```diff
@@ -159,3 +161,63 @@ function joinup_migrate_post_update_more_redirects() {
     }
   }
 }
+
+/**
+ * Switches the ownership of the 'Sharing and reuse of IT solutions' collection.
```

Comment:
Because this file will grow in the next period with a lot of requests to fix content, I decided in ISAICP-3962 (PR https://github.com/ec-europa/joinup-dev/pull/960/files) to add the ticket number in the docblock. This is true **only for joinup_migrate.post_update.php** file, not for other modules. The idea is that the migration module will be removed from the codebase and we will clear even the Git history. But not I'm trying to keep track somehow about these updates. Also, the first word, unlike on other functions/methods, should not end with "s". This is true only for `hook_update_N()` and `hook_post_update_NAME()` functions. So s/Switches/Switch.

Reply:
Ow.. I did not know that :/ I think there are some already to be accepted (merged in UAT) that have been without the ticket number. :/